pages/common/clang-format.md

```diff
@@ -0,0 +1,24 @@
+# clang-format
+
+> A tool to format C/C++/Java/JavaScript/Objective-C/Protobuf/C# code.
+> More information: <https://clang.llvm.org/docs/ClangFormat.html>.
+
+- Format a file and print the result to `stdout`:
+
+`clang-format {{path/to/file}}`
+
+- Format a file inplace:
+
+`clang-format -i {{path/to/file}}`
+
+- Format a file using a predefined coding style:
+
+`clang-format --style={{LLVM|Google|Chromium|Mozilla|WebKit}} {{path/to/file}}`
+
+- Format a file using the `.clang-format` file in the local path:
+
+`clang-format --style=file {{path/to/file}}`
```

Comment:
Missing token syntax? Also, perhaps this would be more consistent with other existing pages?
```suggestion
`clang-format --style={{path/to/.clang-format}} {{path/to/file}}`
```

Reply:
Judging by the [docs](https://clang.llvm.org/docs/ClangFormat.html), `file` is a literal string and does not refer to a filename. When it is present, the program will look for .clang-format file. Correct me if I'm wrong though

Reply:
Yeah, I didn't comment about this because is an special option for `--style`.

Reply:
@sbrl Yeah, I didn't comment about this because is an special option for `--style`.

Reply:
Oh, I see. No worries! The description confused me.